tests/format/lwc/attribute/quotes/__snapshots__/format.test.js.snap

```diff
@@ -0,0 +1,42 @@
+// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing
+
+exports[`quotes.html - {"embeddedLanguageFormatting":"off"} format 1`] = `
+====================================options=====================================
+embeddedLanguageFormatting: "off"
+parsers: ["lwc"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<div double-quoted="{data}"    single-quoted='{data}'
+unquoted={data}>   </div>
+
+<div class="{data}"    class='{data}'
+class={data}>   </div>
+
+=====================================output=====================================
+<div double-quoted="{data}" single-quoted="{data}" unquoted="{data}"></div>
```

Comment:
@kovsu This is what I want to check, `embeddedLanguageFormatting: off` breaks LWC.